sources/controllers/Register.controller.php

```diff
@@ -1258,4 +1265,42 @@ public function action_registerCheckUsername()
 
 		$context['valid_username'] = !$errors->hasErrors();
 	}
+
+	public function action_agrelang()
```

Comment:
Actually, I realized there is a better place for localization-related stuff, so I moved it there. I had to move a bit of other stuff around, but I think it looks nicer now... maybe. :stuck_out_tongue: